src/client/styles/custom.less

```diff
@@ -78,3 +79,17 @@
 @facebook-background-color: #3b5998;
 @twitter-background-color: #55acee;
 @email-background-color: #777777;
+
+@twitter-link-color: @blue-deepSkyBlue;
+
+@modal-background-color: rgba(242, 242, 242, 0.97);
+
+// z-indexs
+@react-modal-z-index: 1400; // used for post image gallery
+@ant-layout-header-z-index: 1050;
+@ant-popup-z-index: 1030;
+@ant-tooltip-z-index: @ant-popup-z-index;
```

Comment:
Added popover and tooltip z-index's here as separate for now incase we would need to change one or the other, maybe if we have a case where we need a tooltip inside a popover...